inference-engine/tests/functional/plugin/shared/src/single_layer_tests/loop.cpp

```diff
@@ -53,7 +53,7 @@ namespace LayerTestsDefinitions {
 
     void LoopTest::SetUp() {
         SKIP_IF_CURRENT_TEST_IS_DISABLED()
-        SetRefMode(LayerTestsUtils::IE);
+        SetRefMode(LayerTestsUtils::INTERPRETER);
         bool execute_first_iteration;
         bool is_body_condition_const;
         bool body_condition; // works only if is_body_condition_const ==
```

Comment:
I had asked you to move Loop creation to the builders from setup before (The same comment for TI). Could you please do it?

Reply:
Not in this PR, have no time for it, not a blocker.